src/main/java/org/sonar/plugins/github/IssueComparator.java

```diff
@@ -32,6 +32,7 @@ public int compare(Issue left, Issue right) {
     if (left == right) {
       return 0;
     }
+    System.out.println("Sample issue");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS106)